### PR TITLE
Add redbelly main, dev, test and tge networks

### PR DESCRIPTION
**Pull request by ekampreet-block8**
### Description

- Add chain specification for Redbelly Mainnet
- Add chain specification for Redbelly Devnet
- Add chain specification for Redbelly Testnet
- Add chain specification for Redbelly Tgenet

### CODE REVIEW

**Comment by namanB8** [comment]
@yash-block8 kindly review

File: _data/chains/eip155-151.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Redbelly Network Mainnet",
+    "shortName": "rbn",
+    "chain": "RBN",
+    "chainId": 151,
+    "networkId": 151,
+    "rpc": [],
+    "faucets": [],
+    "infoURL": "https://www.redbelly.network",
```
Comment (yash-block8) [inline _data/chains/eip155-151.json]: Should we have www? its mostly depprecated now

**Comment by yash-block8** [comment]
Please see if we can update some and add mandatory fields to satisfy the chain spec schema definition here => https://github.com/UCRegistry/chain-registry/blob/master/docs/chain-specification-standard.md

You can use https://www.jsonschemavalidator.net/ with the schema file on the left and the json document on the right. Currently it shows the following errors:

 **Found 2 error(s)
Message:
Invalid type. Expected String but got Integer.
Schema path:
#/properties/networkId/type
Message:
Required properties are missing from object: interface, coins, custom.
Schema path:
#/required**

**Comment by ekampreet-block8** [comment]
> Please see if we can update some and add mandatory fields to satisfy the chain spec schema definition here => https://github.com/UCRegistry/chain-registry/blob/master/docs/chain-specification-standard.md
> 
> You can use https://www.jsonschemavalidator.net/ with the schema file on the left and the json document on the right. Currently it shows the following errors:
> 
> **Found 2 error(s) Message: Invalid type. Expected String but got Integer. Schema path: #/properties/networkId/type Message: Required properties are missing from object: interface, coins, custom. Schema path: #/required**

Using the schema present here inside the repo
https://github.com/redbellynetwork/redbelly-chains/blob/master/tools/schema/chainSchema.json

And validated the same using tool present in the repo
https://github.com/redbellynetwork/redbelly-chains/blob/master/tools/schemaCheck.js

File: _data/chains/eip155-151.json (hunk 1 above)
Comment (ekampreet-block8) [reply _data/chains/eip155-151.json]: @yash-block8 removed www. from the links.

File: _data/chains/eip155-151.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Redbelly Network Mainnet",
+  "shortName": "rbn",
+  "chain": "RBN",
+  "chainId": 151,
+  "networkId": 151,
+  "rpc": [],
```
Comment (gulatinavneet) [inline _data/chains/eip155-151.json:7]: Can these be updated in the future?

**Comment by gulatinavneet** [comment]
@ekampreet-block8 Can we please add some description to the PR around what changes were made.

File: _data/chains/eip155-151.json (hunk 2 above)
Comment (ekampreet-block8) [reply _data/chains/eip155-151.json:7]: Yes we can raise another PR in the future to update these.

I have seen many chains have not added any rpc.
https://github.com/ethereum-lists/chains/blob/03c60edaab42581c16692f4f54b024ffe7efa298/_data/chains/eip155-9.json#L4

Also in this PR a chain has removed it's rpc endpoints.
https://github.com/ethereum-lists/chains/pull/3157/files

Comment (namanB8) [reply _data/chains/eip155-151.json:7]: @gulatinavneet I am marking this conversation as resolved it answers your question.

**Review by namanB8 — APPROVED** [review]
LGTM

**Pull request merged**